index.html

```diff
@@ -4,46 +4,52 @@
   <title>Ayurveda</title>
 </head>
 <body>
-
+<img src="https://herbandpepper.files.wordpress.com/2014/12/cropped-cookingtools-toddquackenbush-unsplash.jpg" alt="Herbs and spices on countertop"/>
 <h1>Ayurveda</h1>
 <h2>The Science of Living</h2>
-
-<p>Ayurveda is the ancient Indian system of healing and well-being. Ayurveda states that balance is the key to a healthy body, mind, and spirit. That balance can be found first by evaluating a person's dosha, or disposition, and then introducing balance through foods, exercises, and practices that temper what's excessive and encourage what's subdued. Although it's best to be examined by an Ayurvedic doctor to determine your dispostion, you can make an educated assumption by evaluating yourself. Most people are a mix of two dosha, with one being more dominant than the other.</p>
-
-<table border=2>
-  <tr>
-    <th>Dosha</th>
-    <th>Governing Element</th>
-    <th>Physical Attributes</th>
-    <th>Mental Attributes</th>
-    <th>Emotional Attributes</th>
-  </tr>
-
-  <tr>
-    <td>Vata</td>
-    <td>Air</td>
-    <td>Natually thin body, dry skin, poor digestion, tends to be cold</td>
-    <td>Fast-moving thoughts, creative</td>
-    <td>Tends towards anxiety</td>
-  </tr>
-
-  <tr>
-    <td>Pitta</td>
-    <td>Fire</td>
-    <td>Naturally muscular, reddish skin, aggressive appetite, tends to be hot</td>
-    <td>Sharp intellect, competitive</td>
-    <td>Tends towards anger</td>
-  </tr>
-
- <tr>
-    <td>Kapha</td>
-    <td>Water/Earth</td>
-    <td>Naturally plump, oily skin, hearty digestion, tends to sweat easily</td>
-    <td>Slow and methodical thinking</td>
-    <td>Tends towards depression</td>
-  </tr>
-
-</table>
-
+<p>written by <a href="contact.html">Ellie Dori</a></p>
+
+<section>
+  <p>Ayurveda is the ancient Indian system of healing and well-being. Ayurveda states that balance is the key to a healthy body, mind, and spirit. That balance can be found first by evaluating a person's <i>dosha</i>, or disposition, and then introducing balance through food, exercise, and daily habits that temper what's excessive and encourage what's subdued. Although it's best to be examined by an Ayurvedic doctor to determine your disposition, you can make an educated assumption by evaluating yourself. Most people are a mix of two <i>doshas</i>, with one being more dominant than the other.</p>
```

Comment:
Try to keep styling in the CSS files, <i> can be replaced with a <span class="emphasis"> or some such.